packages/messages/src/domains/chat.js

```diff
@@ -0,0 +1,21 @@
+const { Schema, ObjectId } = require('mongoose');
+
+const connection = require('../drivers/mongodb/connection');
+
+const User = require('./user');
+const Message = require('./message');
+
+const Chat = new Schema({
+  hostId: { type: ObjectId, required: true, ref: User },
+  customerId: { type: ObjectId, required: true, ref: User },
+  messageId: { type: ObjectId, required: true, ref: Message },
+  place: {
+    placeId: { type: ObjectId, required: true },
```

Comment:
can we rename `placeId` to just `id`?
```suggestion
    id: { type: ObjectId, required: true },
```